charts/netdata/templates/ingress.yaml

```diff
@@ -26,8 +27,18 @@ spec:
       http:
         paths:
           - path: {{ $ingressPath }}
+            {{- if and (ne $apiVersion "extensions/v1beta1") (ne $apiVersion "networking.k8s.io/v1beta1") }}
```

Comment:
perhaps make `isOldAPI` var and use it?

```
if not isOldAPI
 ...
else
 ...
```

At the moment we need not complex _ifs_ here, but only old/not old

Reply:
It might be more difficult to maintain because it might be networking.k8s.io/v2 someday and then you'll have 2 old APIs then more variables and then using explicit strings in condition is more descriptive I think.

Reply:
> it might be networking.k8s.io/v2 someday

Or might not be. Your point is valid, but at the moment the condition is simple: is `networking.k8s.io/v1` or not.

When we have `networking.k8s.io/v2` we no longer support `extensions/v1beta1`, etc.